stats.py

```diff
@@ -521,7 +523,7 @@ def handle_files(files):
     for f in files:
         if f.count("/") == 2:
             solutions.append(tuple(f.split("/")))
-        elif f.count("/") == 1:
+        elif f.count("/") == 1 and f.startswith("./"):
```

Comment:
is this for cases like `ProblemX/Lang` ?

Reply:
This conditional it's be used to build a entire problem. Let's say: `stats.py -b --files ./Problem001` would build all the solutions for Problem001. I though that this was working already, but I cannot make work before. So I write this in the last PR.

But this generated a bug because counting only the slashes it is ambiguous, so I explicitly check the syntax if starts with dot slash.

Reply:
Alright ! thanks for clarifying